libs/contract/src/lib/contract/+state/contract.service.ts

```diff
@@ -52,7 +52,7 @@ export class ContractService extends CollectionService<ContractState> {
   }
 
 
-  async addNegotiation(contractId: string, contract: Partial<Negotiation>) {
+  async addNegotiation(contractId: string, contract: Partial<Negotiation>, shouldUpdateSale = true) {
```

Comment:
remove `shouldUpdateSale`
```suggestion
  async addNegotiation(contractId: string, contract: Partial<Negotiation>) {
```